addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1317,7 +1317,7 @@ var SnippetsMenu = Widget.extend({
             if ($oeStructure.length && !$oeStructure.children().length && this.$snippets) {
                 // If empty oe_structure, encourage using snippets in there by
                 // making them "wizz" in the panel.
-                this._updateRightPanelContent({content: [], tab: this.tabs.BLOCKS});
+                this._activateBlocksTab();
```

Comment:
You should consider the async part you introduce with this, only call odooBounce after the panel actually switched.